hw/ip/otp_ctrl/doc/dv/index.md

```diff
@@ -24,47 +24,65 @@ OTP_CTRL testbench has been constructed based on the [CIP testbench architecture
 * Interrupts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}})
-* Alerts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}})
+* Alerts ([`alert_esc_if`]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}})
 * Devmode ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}})
 
 ### Common DV utility components
 The following utilities provide generic helper tasks and functions to perform activities that are common across the project:
 * [dv_utils_pkg]({{< relref "hw/dv/sv/dv_utils/README.md" >}})
 * [csr_utils_pkg]({{< relref "hw/dv/sv/csr_utils/README.md" >}})
 
-### Compile-time configurations
-[list compile time configurations, if any and what are they used for]
-
 ### Global types & methods
 All common types and methods defined at the package level can be found in
 `otp_ctrl_env_pkg`. Some of them in use are:
 ```systemverilog
-[list a few parameters, types & methods; no need to mention all]
+  parameter uint SCRAMBLE_DATA_SIZE = 64;
+  parameter uint SCRAMBLE_KEY_SIZE  = 128;
+  parameter uint NUM_ROUND          = 31;
+
+  typedef enum bit [2:0] {
+    OtpNoError,
+    OtpMacroError,
+    OtpMacroEccCorrError,
+    OtpMacroEccUncorrError,
+    OtpMacroWriteBlankError,
+    OtpAccessError,
+    OtpCheckFailError,
+    OtpFsmStateError
+  } otp_err_code_e;
+
+  typedef virtual mem_bkdr_if #(.MEM_ECC(1)) mem_bkdr_vif;
 ```
+
 ### TL_agent
 OTP_CTRL testbench instantiates (already handled in CIP base env) [tl_agent]({{< relref "hw/dv/sv/tl_agent/README.md" >}})
 which provides the ability to drive and independently monitor random traffic via
 TL host interface into OTP_CTRL device.
 
-### UVC/agent 1
-[Describe here or add link to its README]
+### Alert agents
+OTP_CTRL testbench instantiates (already handled in CIP base env) two [alert_agents]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}):
+fatal_check_alert and fatal_macro_alert.
+The alert_agents provide the ability to drive and independently monitor alert handshakes via
+alert interfaces in OTP_CTRL device.
 
-### UVC/agent 2
-[Describe here or add link to its README]
+### OTP_CTRL IF
+OTP_CTRL design has specific inputs and outputs to communicate with other IPs including LC_CTRL, OTBN, SRAM, FLASH etc.
+This interface is created to initialize, use simple task to drive, and use assertions to monitor these signals.
 
 ### UVM RAL Model
 The OTP_CTRL RAL model is created with the [`ralgen`]({{< relref "hw/dv/tools/ralgen/README.md" >}}) FuseSoC generator script automatically when the simulation is at the build stage.
 
 It can be created manually by invoking [`regtool`]({{< relref "util/reggen/README.md" >}}):
 
 ### Reference models
-[Describe reference models in use if applicable, example: SHA256/HMAC]
+The OTP_CTRL's utilizes [PRESENT]({{< relref "hw/ip/prim/doc/prim_present.md" >}} as the cipher to scramble and protect secrets.
+Thus OTP_CTRL's scoreboard adopted PRESENT's C reference model, resided under `hw/ip/prim/dv/prim_present/` folder, for encryption and decryption purpose.
```

Comment:
very very small nit: maybe try `located in` instead of `resided under`

Reply:
Thank you Udi, updated the word.

Reply:
Hey Udi, just wonder if you plan to have a small README for that folder?

Reply:
Sure - I can take an item to add a quick README for the PRINCE/PRESENT dpi models.
Thanks for mentioning that!

Reply:
Thank you for doing it!